google/cloud/storage/internal/xml_node.h

```diff
@@ -25,6 +25,10 @@ namespace cloud {
 namespace storage_internal {
 GOOGLE_CLOUD_CPP_INLINE_NAMESPACE_BEGIN
 
+std::string EscapeXmlTag(std::string const& val);
```

Comment:
These probably belong in a different header, maybe `xml_escape.h`?

Reply:
Done